redaxo/src/core/lib/error_handler.php

```diff
@@ -152,6 +152,10 @@ public static function handleException($exception)
                 $errPage
             );
 
+            $errPage = preg_replace('@<button id="copy-button" .*?</button>@s', '$0<button id="copy-button" class="clipboard" data-clipboard-text="'.rex_escape(self::getMarkdownReport($exception)).'" title="Copy exception details and system report as markdown to clipboard">
```

Comment:
kann man hier ggf. autom. auf die github issue seite verlinken und dort bereits den systembericht reinposten?

Reply:
Wüsste spontan nicht, wie man den Issuetext vorbelegen kann (außer über die statischen Issue Templates).
Hinzu kommt, dass man das Feature ja auch nicht nur für das Hauptrepo nutzen möchte, sondern zum Beispiel auch bei FOR-Addons etc. Oder man möchte ein vorhandenes Issue ergänzen, oder in einen PR eine Exception melden, etc.
Daher möchte ich diesen Copy-Button so oder so haben.
Wir können aber gerne dann natürlich noch schauen, ob man noch was automatisierteres hinbekommt.